windows-active-directory-mixin/README.md

```diff
@@ -0,0 +1,154 @@
+# Windows Active Directory mixin
+The Windows Active Directory mixin is a set of configurable Grafana dashboards and alerts.
+
+The Windows Active Directory mixin contains the following dashboards:
+
+- Windows Active Directory overview
+- Windows logs
+
+and the following alerts:
+
+- WindowsActiveDirectoryHighPendingReplicationOperations
+- WindowsActiveDirectoryHighReplicationSyncRequestFailures
+- WindowsActiveDirectoryHighPasswordChange
+- WindowsActiveDirectoryMetricsDown
+
+## Windows Active Directory overview
+The Windows Active Directory overview dashboard provides details on alerts, LDAP operations and requests, bind operations, replication traffic, and database operations.
+![Windows Active Directory overview dashboard (LDAP)](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/windows_active_directory_overview_1.png)
+![Windows Active Directory overview dashboard (database)](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/window_active_directory_overview_2.png)
+
+# Windows logs
+The Windows logs dashboard provides details on incoming Windows application, security, and system logs.
+![Windows logs dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/windows_active_directory_logs.png)
+
+Windows logs are enabled by default in the `config.libsonnet` and can be disabled by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+For the selectors to properly work with the Windows logs ingested into your logs datasource, please also include the matching `instance` and `job` labels in the [scrape configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) to match the labels for ingested metrics.
+
+```yaml
```

Comment:
Is this yaml section new? I don't remember it being in the README before

Reply:
This yaml section is just to show what the log snippet would look like for the mixin. Its not new as far as I know and I have been asked to include it in the past.